caf-audit-binding-elasticsearch/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditChannelTest.java

```diff
@@ -15,59 +15,61 @@
  */
 package com.hpe.caf.auditing.elastic;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
 import com.hpe.caf.auditing.AuditCoreMetadataProvider;
 import com.hpe.caf.auditing.AuditEventBuilder;
 import com.hpe.caf.auditing.internal.AuditNewEventFactory;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
-import org.opensearch.client.IndicesClient;
-import org.opensearch.client.RestHighLevelClient;
-import org.opensearch.client.indices.GetIndexTemplatesRequest;
-import org.opensearch.client.indices.GetIndexTemplatesResponse;
-import org.opensearch.client.indices.IndexTemplateMetadata;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import static org.mockito.ArgumentMatchers.any;
 import org.mockito.Mockito;
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.opensearch.client.opensearch.indices.GetIndexTemplateRequest;
+import org.opensearch.client.opensearch.indices.GetIndexTemplateResponse;
+import org.opensearch.client.opensearch.indices.OpenSearchIndicesClient;
+import org.opensearch.client.opensearch.indices.get_index_template.IndexTemplateItem;
 
 public class ElasticAuditChannelTest {
 
```

Comment:
I don't think this class tests anything, I can delete the setup method and set `mockOpenSearchClient` to null and it will still pass all the tests?